arangodb-net-standard/Serialization/IApiClientSerializationOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace ArangoDBNetStandard.Serialization
+{
+    /// <summary>
+    /// Define the options.
+    /// </summary>
+    public interface IApiClientSerializationOptions
```

Comment:
Do we really need an interface for this? The class `ApiClientSerializationOptions` declared elsewhere should be enough. It can be inherited if consumers needs special accessors behavior. I'm just curious.